oat-sa/extension-tao-itemqti-pci
# Feature/nec 21/new math entry symbols

**Related to task:** 
https://oat-sa.atlassian.net/browse/NEC-21

**Description:**
Adding more LaTeX symbols

**How to test:**
1. Setup a test with at least one item with a math entry interaction
2. Setup a delivery with the test
3. Run the test and check if symbols are working properly
